web/docs/advanced/apis.md

```diff
@@ -136,48 +136,23 @@ For example, if your app is running at `https://example.com` then from the above
 
 To use the API from your client, including with auth support, you can import the Axios wrapper from `wasp/client/api` and invoke a call. For example:
 
-<Tabs groupId="js-ts">
-<TabItem value="js" label="JavaScript">
-
-```jsx title="src/pages/SomePage.jsx"
-import React, { useEffect } from "react";
-import { api } from "wasp/client/api";
+```tsx title="src/pages/SomePage.tsx" auto-js with-hole
+import React, { useEffect } from 'react'
```

Comment:
I think your formatter removes `;` from the end? I think we actually use them on purpose in general but also in the whole docs. So let's try to keep them for now, unless we decide we don't want to have them. And we can make decisions if we want to use them or not when we will be defining linting and formatting for the whoel wasp proejct as we have in that other issue.